src/ParallelAlgorithms/Interpolation/CMakeLists.txt

```diff
@@ -30,21 +30,29 @@ add_dependencies(
   module_GlobalCache
   )
 
+# Some of these dependencies should be removed, see issue
+# https://github.com/sxs-collaboration/spectre/issues/3561
 target_link_libraries(
   ${LIBRARY}
   PUBLIC
   ApparentHorizons
   DataStructures
-  DgSubcell
   Domain
-  DomainStructure
   ErrorHandling
-  EventsAndTriggers
-  GSL::gsl
+  GeneralRelativitySolutions
```

Comment:
Why isn't GeneralRelativitySolutions in INTERFACE?

Reply:
The KerrHorizon.cpp includes it through KerrHorizon.hpp, so it's needed to compile the lib (see my comment in your docs PR)